CMakeLists.txt

```diff
@@ -136,3 +145,8 @@ CONFIGURE_FILE(cmake/mfmgConfig.cmake.in
 INSTALL(FILES ${CMAKE_BINARY_DIR}/cmake/mfmgConfig.cmake
   DESTINATION ${CMAKE_INSTALL_PREFIX}/lib/cmake/mfmg
   )
+
+INSTALL(DIRECTORY ${PROJECT_SOURCE_DIR}/include/ DESTINATION include
+  FILES_MATCHING PATTERN "*.hpp")
```

Comment:
This is necessary because you removed `PUBLIC_HEADER "${MFMG_HEADERS}"` when setting the target properties and thus `PUBLIC_HEADER DESTINATION ${CMAKE_INSTALL_PREFIX}/include` doesn't do anything when you install `mfmg`. Normally `INSTALL(TARGET mfmg)` should be enough to install `mfmg`

Reply:
It didn't for me.